echidna/scripts/dump_shrink.py

```diff
@@ -0,0 +1,100 @@
+""" Example shrinking script
+
+This script:
+  * Reads in spectrum from hdf5
+  * shrinks spectrum with user input pars and low high values
+  * Saves shrunk spectrum to given output or, by default, to the same
+    directory with _shrink added to the filename.
+
+Examples:
+  To shrink hdf5 file ``example.hdf5`` that has dimensions energy_mc and
+  radial3_mc to a range in energy of 1. to 4. MeV and radial3 from 0. to 0.5
+  and save the spectrum to ``example2.hdf5`` then the following command is
+  required::
+
+    $ python dump_shrink.py -i /path/to/example.hdf5 -o /path/to/example2.hdf5
+      -p energy_mc radial3_mc -l 1. 0. -u 4. 0.5
+
+  This will create the shrunk hdf5 file ``/path/to/example2.hdf5``.
+
+"""
+
+from echidna.output import store
+import echidna.utilities as utilities
+import os
+
+_logger = utilities.start_logging()
+
+
+def main(args):
+    """Smears energy and dumps spectra.
+
+    Args:
+      args (Namespace): Container for arguments. See
+        >>> python dump_smeared_energy.py -h
```

Comment:
This line doesn't get formatted correctly as a code-block in the built docs. I think you just need to add a double colon `::` after " See" and then leave a blank line before indenting the code-block.

Also this is something you would do from a standard terminal prompt, not from an interactive python session, so probably doesn't need the `>>>` prompt.